adapters/triplelift_native/triplelift_native.go

```diff
@@ -0,0 +1,195 @@
+package triplelift_native
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/mxmCherry/openrtb"
+	"github.com/prebid/prebid-server/adapters"
+	"github.com/prebid/prebid-server/errortypes"
+	"github.com/prebid/prebid-server/openrtb_ext"
+	"net/http"
+)
+
+type TripleliftNativeAdapter struct {
+	endpoint string
+	extInfo  TripleliftNativeExtInfo
+}
+
+type TripleliftInnerExt struct {
+	Format int `json:"format"`
+}
+
+type TripleliftRespExt struct {
+	Triplelift TripleliftInnerExt `json:"triplelift_pb"`
+}
+
+type TripleliftNativeExtInfo struct {
+	// Array is used for deserialization.
+	PublisherWhitelist []string `json:"publisher_whitelist"`
+
+	Endpoint string `json:"endpoint"`
+
+	// Map is used for optimized memory access and should be constructed after deserialization.
+	PublisherWhitelistMap map[string]bool
+}
+
+func getBidType(ext TripleliftRespExt) openrtb_ext.BidType {
+	return openrtb_ext.BidTypeNative
```

Comment:
Can I ask what's the purpose of passing the `TripleliftRespExt` to this function here?

Reply:
There's no point right now, but in the future we might want to inspect that object.